src/main/java/taxi/controller/driver/AddDriverController.java

```diff
@@ -24,7 +24,8 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp)
     public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOException {
         String name = req.getParameter("name");
         String licenseNumber = req.getParameter("license_number");
-        Driver driver = new Driver(name, licenseNumber);
+        String login = req.getParameter("login");
+        Driver driver = new Driver(name, licenseNumber,login);
         driverService.create(driver);
         resp.sendRedirect(req.getContextPath() + "/drivers/add");
```

Comment:
```suggestion
        resp.sendRedirect(req.getContextPath() + "/drivers");
```
after you will redirect to page with all drivers and see added driver